relay-server/src/services/processor.rs

```diff
@@ -504,6 +504,30 @@ impl<'a> ProcessEnvelopeState<'a> {
         self.remove_event();
         self.managed_envelope.reject_event(outcome);
     }
+
+    fn forward_unsampled_profiles(&self, global_config: &GlobalConfig) -> bool {
```

Comment:
```suggestion
    /// Determines whether profiles that would otherwise be dropped by dynamic sampling should be kept.
    fn forward_unsampled_profiles(&self, global_config: &GlobalConfig) -> bool {
```